packages/browser/__tests__/ClientAuthentication.spec.ts

```diff
@@ -22,10 +22,15 @@
 // Required by TSyringe:
 import "reflect-metadata";
 import {
+  IIssuerConfig,
   mockStorageUtility,
   StorageUtility,
+  USER_SESSION_PREFIX,
 } from "@inrupt/solid-client-authn-core";
 import { mockStorage } from "@inrupt/solid-client-authn-core/dist/storage/__mocks__/StorageUtility";
+import { Response } from "cross-fetch";
+import { JSONWebKey } from "jose";
+import { signJwt } from "@inrupt/oidc-client-ext";
```

Comment:
Just wondering... Shouldn't this be a relative import?

Reply:
I'm not 100% sure, but I think Lerna deals with that under the cover, that's how I understand `lerna bootstrap` at least. Each code base acts as if it was independent, and it's the monorepo tooling that stitches it together.